frontend/src/core/comments/actions.js

```diff
@@ -0,0 +1,38 @@
+/* @flow */
+
+import NProgress from 'nprogress';
+
+import api from 'core/api';
+import * as notification from 'core/notification';
+import * as history from 'modules/history';
+import * as teamcomments from 'modules/teamcomments';
+
+
+export function addComment(
+    entity: number,
+    locale: string,
```

Comment:
`entity` and `locale` are only needed when adding a team comment, and `translation` is only needed when adding a translation comment, right? If so, why always require `entity` and `locale`? Could it work to only require either `entity` + `locale` or `translation`?

Reply:
Within that function the calls to `get` require `locale` and `entity`

Reply:
Right, we need to update history.

I'd be nice to make them optional in `AddCommentsForm`.